.ci/validation/src/index.ts

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
-import fs from "fs";
-import Ajv from "ajv";
+import fs from "node:fs";
+import Ajv from "ajv/dist/2020";
```

Comment:
I updated the ajv import to use draft 2020-12 of json schema to support newer features such as `unevaluatedProperties`